content/hi/docs/reference/glossary/cluster-architect.md

```diff
@@ -0,0 +1,17 @@
+---
+title: क्लस्टर आर्किटेक्ट (Cluster Architect)
+id: cluster-architect
+date: 2018-04-12
+full_link: 
+short_description: >
+ क्लस्टर आर्किटेक्ट क्लस्टर के बुनियादी ढांचे को डिजाइन करने के लिए जिम्मेदार है, जिसमें एक या अधिक Kubernetes क्लस्टर शामिल हो सकते हैं।
+ 
+aka: 
+tags:
+- user-type
+---
+क्लस्टर आर्किटेक्ट क्लस्टर के बुनियादी ढांचे को डिजाइन करने के लिए जिम्मेदार है, जिसमें एक या अधिक कुबेरनेट क्लस्टर शामिल हो सकते हैं।

```

Comment:
```suggestion
एक व्यक्ति जो बुनियादी ढांचे को रचना करता है जिसमें एक या एक से अधिक कुबेरनेट्स क्लस्टर शामिल होते हैं।

```
same as above

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix